README.md

```diff
@@ -1,72 +1,92 @@
+# Catalog of my things
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description of the project.
-
+> In this project, we created console app that helps users to keep a record of different types of things we won: books, music albums, movies, and games. Everything is stored on a JSON file and can be retrieved even after the user exit from the application.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- **Ruby**
 
-## Live Demo (if available)
+## Setup
 
-[Live Demo Link](https://livedemo.com)
+- Install Ruby in local envinronment.
+- Run program(irb) using the command line
 
+## Prerequisites
+- Understand Ruby basic system and linux
+- No linter errors
 
+## Video link
+- [Video Presentation](https://drive.google.com/file/d/1M8U-6YQBTusXeqdgKs2aOFI9UGJoW1qX/view?usp=sharing)
```

Comment:
- [ ] The maximum length of the video should be 5 minutes. Currently, the length of your video is `7.17 minutes`  Kindly shorten your video and not more than 5 minutes. :camera: 

- [ ] All the members should appear in the video and talk about the project. I can see only two of you in the video. :+1: 

- [ ] You should highlight a piece of code that you find interesting or proud of making. :+1:

Reply:
@mahbubul14 We will do it now